tests/testsuite.py

```diff
@@ -1155,6 +1155,19 @@ def test_hashed_first_over2_same_value(self):
         self.assertIsNone(display_hint)
 
 
+@unittest.skipIf(boost_version < (1, 71), 'implemented in boost 1.71 and later')
```

Comment:
Was internal structure of wave objects different before boost 1.71?

Reply:
Perhaps. One of the old builds was failing - 1.55 or 1.65. Since this printer is new I decided to limit support to something recent.

Reply:
:ok_hand: